lib/online_order.rb

```diff
@@ -0,0 +1,84 @@
+require "csv"
+require_relative "order"
+require_relative "customer"
+
+module Grocery
+  class OnlineOrder < Order
+    attr_reader :id, :products, :customer, :shipping, :online_orders
+    @@online_orders = []
+    Customer.all
+
+    def initialize(id, products, customer, shipping = :pending)
+      @id = id
+      @products = products
```

Comment:
Instead of saving `@id` and `@products` here, you could call `super(id, products)`, which will run the `initialize` method on `Order` and save them for you.